clients/src/main/java/org/apache/kafka/common/protocol/types/Field.java

```diff
@@ -46,6 +46,17 @@ public Field(String name, Type type) {
         this(name, type, null, false, null);
     }
 
+    @Override
+    public String toString() {
+        return "Field{" +
+                "name='" + name + '\'' +
+                ", docString='" + docString + '\'' +
```

Comment:
the docString field maybe very long so how about putting it to last？

Reply:
good suggestion. will address it.